src/styles/blocks/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    align-self: center;
+  }
+}
+
+.nav {
```

Comment:
.nav this is a block and perhaps it makes sense to move it into a separate file "nav.scss"

Reply:
agree. Each BEM-block should be located in a separate file

Reply:
each block should in the separate file